lms/djangoapps/grades/models.py

```diff
@@ -34,60 +33,62 @@ class BlockRecordList(tuple):
     An immutable ordered list of BlockRecord objects.
     """
 
-    def __new__(cls, blocks):
+    def __new__(cls, blocks, course_key):  # pylint: disable=unused-argument
         return super(BlockRecordList, cls).__new__(cls, tuple(blocks))
```

Comment:
Side note: What is the `__new__()` method doing, above?  `super(BlockRecordList, cls)` is `tuple`, so it looks like it's calling `tuple.__new__(BlockRecordList, tuple(blocks))`, which is like calling `tuple(BlockRecordList, tuple(blocks))`.  I must be misunderstanding something because it seems like that should just not work.

Can we just do away with this method?